knative-kubecon/README.md

```diff
@@ -1,3 +1,219 @@
+we need from Knative installed, run the following script:
+
+```bash
+$ git clone git@github.com:openshift-cloud-functions/knative-operators.git
+$ ./knative-operators/etc/scripts/install.sh
+```
+
+> This script is destructive towards the 'knative' profile in minishift.
+
+After this script exits without any errors, your cluster will be setup and ready to go. Next, let's make `oc`
+(OpenShift's CLI) available in the current terminal:
+
+```bash
+$ eval $(minishift oc-env)
+```
+
+There we are, let's get crackin'.
+
+## 1. The Build component
+
+The Build component in Knative is not so much a utility to build images themselves. It rather provides primitives,
+to be able to string together the tools you want to do your image build. In a sense, it's an abstraction layer above
+all the tools out there to build an image. In our case, the most prominent example is OpenShift's own build capability,
+so this will show you, how we can implement a Knative Build by the means of an OpenShift Build.
+
+Knative provides a mechanism called **BuildTemplates**, where you define a blueprint for a build, which contains the
+arguments that that build might need to do its job of building the image in the end. An example of such a template can
+be seen below. This template allows building images through Knative Build based on OpenShift's Build capability, so you
+can use the tools you're used to while taking full advantage of Knative Build on top of it.
+
+```yaml
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: openshift-builds
+spec:
+  parameters:
+  - name: IMAGE
+    description: The name of the image to push
+  - name: NAME
+    description: Build configuration name
+  - name: IMAGE_STREAM
+    description: The image stream to use as input for the build
+  - name: TO_DOCKER
+    description: Push the image to a Docker repository or not (true by default)
+    default: "false"
+  - name: DIRECTORY
+    description: The directory containing the app
+    default: /workspace
+  - name: OC_BUILDER_IMAGE
+    description: The name of the builder image to use
+    default: docker.io/vdemeester/kobw-builder:0.0.1
+  steps:
+  - name: kobw-create-or-update
+    image: "${OC_BUILDER_IMAGE}"
+    args: ["create", "--name=${NAME}", "--image=${IMAGE}", "--image-stream=${IMAGE_STREAM}", "--to-docker=${TO_DOCKER}", "."]
+    workingDir: "${DIRECTORY}"
+    env:
+      - name: NAMESPACE
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.namespace
+  - name: kobw-run
+    image: "${OC_BUILDER_IMAGE}"
+    args: ["run", "--name=${NAME}"]
+    env:
+      - name: NAMESPACE
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.namespace
+```
+
+To achieve the desired effect, this template wraps an OpenShift Build entity to perform the build of the desired application.
+To "install" that template, we need to `oc apply` it by:
+
+```bash
+$ oc apply -f build/000-build-template.yaml
```

Comment:
what should happen here ? 

e.g. mention that this will no create pods or so - it's more config ... or , well... template